src/core/prompts/__tests__/__snapshots__/system.test.ts.snap

```diff
@@ -392,19 +392,47 @@ Always adhere to this format for the tool use to ensure proper parsing and execu
 # Tools
 
 ## read_file
-Description: Request to read the contents of a file at the specified path. Use this when you need to examine the contents of an existing file you do not know the contents of, for example to analyze code, review text files, or extract information from configuration files. The output includes line numbers prefixed to each line (e.g. "1 | const x = 1"), making it easier to reference specific lines when creating diffs or discussing code. Automatically extracts raw text from PDF and DOCX files. May not be suitable for other types of binary files, as it returns the raw content as a string.
+Description: Request to read the contents of a file at the specified path. Use this when you need to examine the contents of an existing file you do not know the contents of, for example to analyze code, review text files, or extract information from configuration files. The output includes line numbers prefixed to each line (e.g. "1 | const x = 1"), making it easier to reference specific lines when creating diffs or discussing code. By specifying start_line and end_line parameters, you can efficiently read specific portions of large files without loading the entire file into memory. Automatically extracts raw text from PDF and DOCX files. May not be suitable for other types of binary files, as it returns the raw content as a string.
 Parameters:
 - path: (required) The path of the file to read (relative to the current working directory /test/path)
+- start_line: (optional) The starting line number to read from (1-based). If not provided, it starts from the beginning of the file.
+- end_line: (optional) The ending line number to read to (1-based, inclusive). If not provided, it reads to the end of the file.
```

Comment:
Does it read to the end of the file? Or is it limited by the max lines to read?

Reply:
i will limited by the max lines to read, if not specify read range